direct/src/dist/FreezeTool.py

```diff
@@ -41,9 +41,6 @@
 if sys.version_info >= (3, 0):
```

Comment:
We don't need to branch here anymore and can just set `startupModules` directly.

Reply:
Ok ! I will set it